openfl/interface/workspace.py

```diff
@@ -203,9 +224,7 @@ def import_(archive):
         check_call([
             executable, '-m', 'pip', 'install', '--upgrade', 'pip'],
             shell=False)
-        check_call([
-            executable, '-m', 'pip', 'install', '-r', 'requirements.txt'],
-            shell=False)
+        _import_whls_to_venv()
```

Comment:
This could break if different collaborators are using a different python version than the aggregator. Because this assumes that all dependencies are captured in `requirements.txt` anyway, I would recommend falling back to the original behavior (i.e. `pip install -r requirements.txt`) in the case that the import fails